Libraries/LibJS/Runtime/ArrayPrototype.cpp

```diff
@@ -400,4 +401,46 @@ Value ArrayPrototype::last_index_of(Interpreter& interpreter)
     return Value(-1);
 }
 
+Value ArrayPrototype::fill(Interpreter& interpreter)
+{
+    auto* array = array_from(interpreter);
+    if (!array)
+        return {};
```

Comment:
`Array.prototype.fill` should be generic and work on any Object, not just Arrays.